tests/keras_tests/feature_networks_tests/feature_networks/const_representation_test.py

```diff
@@ -81,6 +78,38 @@ def compare(self, quantized_model, float_model, input_x=None, quantization_info=
         self.unit_test.assertTrue(np.isclose(cs, 1), msg=f'fail cosine similarity check:{cs}')
 
 
+class ConstRepresentationMatMulTest(ConstRepresentationTest):
+    def __init__(self, unit_test, layer, const, is_list_input=False, input_reverse_order=False, use_kwrags=False,
+                 transpose_a=False, transpose_b=False, input_shape=(32, 32, 16)):
+        super(ConstRepresentationMatMulTest, self).__init__(unit_test=unit_test, input_shape=input_shape, layer=layer,
+                                                            const=const, is_list_input=is_list_input,
+                                                            input_reverse_order=input_reverse_order,
+                                                            use_kwrags=use_kwrags)
+        self.transpose_a = transpose_a
+        self.transpose_b = transpose_b
+
+    def create_networks(self):
```

Comment:
Consider converting this test to a single test (less args...) that tests all combinations, like
`matmul(x, const)`
`matmul(x, b=const)`
`matmul(a=x, b=const)`
`matmul(x, const, transpose_b=True)`
`matmul(x, const, False, True)`
etc.
const may be a `tf.Variable`, `np.ndarray`, list or tuple of floats\ints